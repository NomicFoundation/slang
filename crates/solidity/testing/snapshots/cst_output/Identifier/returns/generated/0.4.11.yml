# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Source: >
  1  │ returns                                                                          │ 0..7

Errors: # 1 total
  - >
    Error: Expected something else.
       ╭─[crates/solidity/testing/snapshots/cst_output/Identifier/returns/input.sol:1:1]
       │
     1 │ returns
       · ┬  
       · ╰── Found 'r'.
    ───╯

Tree: null
