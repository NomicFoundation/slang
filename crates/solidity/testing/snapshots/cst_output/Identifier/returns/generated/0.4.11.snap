# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Error: Expected something else.
   ╭─[crates/solidity/testing/snapshots/cst_output/Identifier/returns/input.sol:1:1]
   │
 1 │ returns
   · ┬  
   · ╰── Found 'r'.
───╯

Tree: null
